src/core_plugins/kibana/public/home/components/tutorial/footer.js

```diff
@@ -11,7 +11,7 @@ import {
 
 export function Footer({ url, label }) {
   if (!url) {
-    return;
+    return null;
```

Comment:
what do you think about making url and label required and also pushing this logic externally?